package-lock.json

```diff
@@ -14093,7 +14097,8 @@
         "diagnostic-channel-publishers": {
             "version": "0.3.5",
             "resolved": "https://registry.npmjs.org/diagnostic-channel-publishers/-/diagnostic-channel-publishers-0.3.5.tgz",
-            "integrity": "sha512-AOIjw4T7Nxl0G2BoBPhkQ6i7T4bUd9+xvdYizwvG7vVAM1dvr+SDrcUudlmzwH0kbEwdR2V1EcnKT0wAeYLQNQ=="
+            "integrity": "sha512-AOIjw4T7Nxl0G2BoBPhkQ6i7T4bUd9+xvdYizwvG7vVAM1dvr+SDrcUudlmzwH0kbEwdR2V1EcnKT0wAeYLQNQ==",
+            "requires": {}
```

Comment:
There are some extra changes here in the `pacakge-lock.json`?

Reply:
This happens every once and a while (such as `"dev": true` appear as well).  I haven't been able to find a really good reason why it does, but since it's just an empty {}, it's most likely harmless.